web/gui/custom/README.md

```diff
@@ -129,104 +155,143 @@ If you need to set any other URL as the default Netdata server for all charts th
+-   **relative** number of seconds to now. To show the last 10 minutes of data,
+    `AFTER_SECONDS` must be `-600` (relative to now) and `BEFORE_SECONDS` must
+    be `0` (meaning: now). If you want the chart to auto-refresh the current
+    values, you need to specify **relative** values.
 
 ### Chart sizes
 
 You can set the size of the chart using this:
 
 ```html
 <div data-netdata="unique.id"
-     data-width="WIDTH"
-     data-height="HEIGHT"
-     ></div>
+ data-width="WIDTH"
+ data-height="HEIGHT"
+ ></div>
 ```
 
-`WIDTH` and `HEIGHT` can be anything CSS accepts for width and height (e.g. percentages, pixels, etc).
-Keep in mind that for certain chart libraries, `dashboard.js` may apply an aspect ratio to these.
+`WIDTH` and `HEIGHT` can be anything CSS accepts for width and height (e.g.
+percentages, pixels, etc). Keep in mind that for certain chart libraries,
+`dashboard.js` may apply an aspect ratio to these.
 
-If you want `dashboard.js` to remember permanently (browser local storage) the dimensions of the chart (the user may resize it), you can add: `data-id="SETTINGS_ID"`, where `SETTINGS_ID` is anything that will be common for this chart across user sessions.
+If you want `dashboard.js` to remember permanently (browser local storage) the
+dimensions of the chart (the user may resize it), you can add: `data-id="
+SETTINGS_ID"`, where `SETTINGS_ID` is anything that will be common for this
+chart across user sessions.
 
 ### Netdata server
 
-Each chart can get data from a different Netdata server. You can give per chart the Netdata server using:
+Each chart can get data from a different Netdata server. You can give per chart
+the Netdata server using:
 
 ```html
 <div data-netdata="unique.id"
-     data-host="http://another.netdata.server:19999/"
-     ></div>
+ data-host="http://another.netdata.server:19999/"
+ ></div>
 ```
 
-If you have ephemeral monitoring setup ([More info here](../../../streaming/#monitoring-ephemeral-nodes)) and have no direct access to the nodes dashboards, you can use the following:
+If you have ephemeral monitoring setup ([More info
+here](../../../streaming/#monitoring-ephemeral-nodes)) and have no direct access
+to the nodes dashboards, you can use the following:
 
 ```html
 <div data-netdata="unique.id"
-     data-host="http://yournetdata.server:19999/host/reported-hostname"
-     ></div>
+ data-host="http://yournetdata.server:19999/host/reported-hostname"
+ ></div>
 ```
 
 ### Chart library
 
-The default chart library is `dygraph`. You set a different chart library per chart using this:
+Netdata supports a number of chart libraries. The default is Dygraph, but the
+standard dashboard also uses Gauge.js and EasyPieChart frequently to create
+overviews of essential metrics.
+
+The default chart library is `dygraph`, but you can set a different chart
```

Comment:
You're already stating that the default is dygraph above twice. Review the whole paragraph please.